### ISSUE TITLE
hexo-tag-link-preview updated to 1.3.0, error message will appear when running command.

### ISSUE THREAD

**Issue by carl0schen** [minamo173/hexo-tag-link-preview#79]
hexo-tag-link-preview to 1.3.0, error message will appear when running `hexo s` or `hexo clean` command.
Can you fix this problem? 
Thanks.

Hexo 6.0.0 + hexo-tag-link-preview 1.2.4 > OK
Hexo 5.4.0 + hexo-tag-link-preview 1.2.4 > OK
Hexo 6.0.0 + hexo-tag-link-preview 1.3.0 > NG
Hexo 5.4.0 + hexo-tag-link-preview 1.3.0 > NG
```
carlos@MacBook-Pro-2017 hexo % hexo s                                    
INFO  Validating config
ERROR {
  err: TypeError: Cannot read property 'className' of undefined
      at /Users/carlos/Documents/Hexo/node_modules/hexo-tag-link-preview/index.js:15:45
      at /Users/carlos/Documents/Hexo/node_modules/hexo/lib/hexo/index.js:298:14
      at tryCatcher (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/promise.js:547:31)
      at Promise._settlePromise (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/promise.js:604:18)
      at Promise._settlePromise0 (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/promise.js:729:18)
      at _drainQueueStep (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/async.js:93:12)
      at _drainQueue (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/async.js:86:9)
      at Async._drainQueues (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/async.js:102:5)
      at Immediate.Async.drainQueues [as _onImmediate] (/Users/carlos/Documents/Hexo/node_modules/bluebird/js/release/async.js:15:14)
      at processImmediate (internal/timers.js:439:21)
} Plugin load failed: %s hexo-tag-link-preview
INFO  Start processing
INFO  Hexo is running at http://localhost:4000 . Press Ctrl+C to stop.
```

**Comment by minamo173** [comment]
@carl0schen 
Thanks for reporting!
I will investigate this.

**Comment by minamo173** [comment]
@carl0schen 
Hi, I fixed this problem and released [v1.4.0](https://www.npmjs.com/package/hexo-tag-link-preview/v/1.4.0).
Please try [v1.4.0](https://www.npmjs.com/package/hexo-tag-link-preview/v/1.4.0) and check fixed this error.
Thanks.

**Comment by carl0schen** [comment]
@minamo173 
I have updated to 1.4.0, and after running `hexo s`,  `hexo g` `hexo clean`, it looks fine.
Thanks.
